docs/finalproject/external-projects.md

```diff
@@ -0,0 +1,127 @@
+---
+id: external-projects
+title: External Projects
+sidebar_label: External Projects
+---
+
+At CodeYourFuture we try to find interesting, tech-for-good projects for that stretch the abilities of our students whilst helping an NGO or Charity solve a business problem they are facing.
+
+## Past Projects
+
+Here are some of the projects that we've worked on in the past
+
+- [Over The Line with The Compass Collective](https://over-the-line.uk/)
+
+Additionally, here are some examples of internal projects we've completed
+
+- [Class Planner](https://cyf-class-planner.herokuapp.com/)
+
+You can also find more example project briefs on the Project Brief
+
+## Goals
+
+With the final projects, we’re looking to give our students:
+
+- Experience of collaborating in a team to deliver software
+- An opportunity to solve real world business problems
+- The chance to bring together the different things they’ve learned
+- A real example to talk about in job interviews and guide their interests
+
+and to give our partners:
+
+- A useful piece of software that they can use as-is or build on after the project
+
+More details can be found on the [Intro](./intro) and [Preparation](./prep) pages.
+
+## What does a Product Owner have to do?
+
+To be a good Stakeholder/Product Owner you should
+
+- Be able to attend a weekly/bi-weekly meeting with our teams
+- Be in a position to advise the development of the project
+  - e.g. Should we prioritise Feature A or Feature B?
+- Have realistic expectations of what our students can build in the time given to them
+
+In addition to this, it is very helpful if you can join our internal instant messaging tool (Slack) to speak to our students more regularly. Historically, one of the most common blockers to a successful project has been the quick communication between stakeholders and our students. Students may likely become blocked, spin wheels or go off on a tangent if they have a burning question and can't get it answered.
+
+In the ideal world, you would be able to answer questions within a 0.5-1 working days of the questions being asked by our students. If this might not be possible, let us know and we will select an Internal Product Manager who will be empowered to make decisions on your behalf.
+
+## What is a Good Project?
+
+It's important that the projects that we work on fulfill some base requirements to make sure they go smoothly.
+
+All Final Projects aim to be a [Minimum Viable Product](https://www.agilealliance.org/glossary/mvp/). This means they may be rough around the edges but have all of the core functionality working.
+
+### Functionality
+
+A Good Final Project should have the following qualities
+
+1. Be an interactive website
+
+   - This means a website that a user can interact with to create, read, upload, delete data.
+   - For example, a photo hosting service or blog would fulfil this criteria
```

Comment:
I'm a bit hesitant about the blog example. I would guess that for most third-party stakeholders Wordpress and friends would be much more suited to their needs. (Which we obviously wouldn't be able to do).

Some ideas for alternatives: custom events or Q&A service.

If we're going with a blog example, then maybe we could frame it as "custom blog functionality" or something?